app/models/allocation_version.rb

```diff
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
+
+class AllocationVersion < ApplicationRecord
+  has_paper_trail
+
+  attr_accessor :responsibility
+
+  ALLOCATE_PRIMARY_POM = 0
+  REALLOCATE_PRIMARY_POM = 1
+  ALLOCATE_SECONDARY_POM = 2
+  REALLOCATE_SECONDARY_POM = 3
+  DEALLOCATE_PRIMARY_POM = 4
+  DEALLOCATE_SECONDARY_POM = 5
+
+  USER = 0
+  OFFENDER_MOVEMENT = 1
+
+  enum event: {
+    allocate_primary_pom: ALLOCATE_PRIMARY_POM,
```

Comment:
Enum values are manually assigned in order to prevent problems that arise from using defaults and accidentally changing the order

Reply:
What do you think about adding a comment explaining how to add a new event and why it is done that way - to help others add new ones.

Reply:
👍

Reply:
Done